plugin/adsk/scripts/AEmayaUsdProxyShapeBaseTemplate.mel

```diff
@@ -0,0 +1,249 @@
+global proc AEmayaUsdProxyShapeInfoNew(string $input)
+{
+    textFieldGrp -label `getMayaUsdString("kRootLayer")` -editable 0
+                 -ann `getMayaUsdString("kRootLayerAnn")`
+                 mayaUsdProxyShapeRootLayer;
+
+    textFieldGrp -label `getMayaUsdString("kDefaultPrim")` -editable 0
+                 -ann `getMayaUsdString("kDefaultPrimAnn")`
+                 mayaUsdProxyShapeDefaultPrim;
+
+    AEmayaUsdProxyShapeInfoReplace($input);
+}
+
+global proc AEmayaUsdProxyShapeInfoReplace(string $input)
+{
+    // From the input attribute we simply want the node name.
+    string $nodeName = plugNode($input);
+    string $res[] = `ls -l $nodeName`;
+    string $fullNodeName = $res[0];
+
+    // Call python helpers to get the info we want to display.
+    string $rootLayer   = `python("import AETemplateHelpers; AETemplateHelpers.GetRootLayerName('" + $fullNodeName + "')")`;
+    string $defaultPrim = `python("import AETemplateHelpers; AETemplateHelpers.GetDefaultPrimName('" + $fullNodeName + "')")`;
+
+    textFieldGrp -edit -text $rootLayer mayaUsdProxyShapeRootLayer;
+    textFieldGrp -edit -text $defaultPrim mayaUsdProxyShapeDefaultPrim;
+}
+
+global proc AEmayaUsdProxyShapeBaseTemplate( string $nodeName )
```

Comment:
I created this file by copying the existing "AEmayaUsdProxyShapeTemplate.mel" and renaming this proc (to match the proxy shape base node).